kafka.spec

```diff
@@ -21,7 +21,7 @@ Source4: %{name}.sysconfig
 %if %{build_with_metrics}
 # adding metric specific sources.
 Source6: metrics-graphite-2.2.0.jar
-Source7: kafka-graphite-1.0.5.jar
+Source7: kafka-graphite-1.0.6.jar
```

Comment:
If Kafka 2.6 the location of the metrics registry changed and kafka-graphite-1.0.6 was released for it.

https://cwiki.apache.org/confluence/display/KAFKA/KIP-544
https://github.com/damienclaveau/kafka-graphite/pull/28